### PR TITLE
Add a migration guide for gss

### PR DESCRIPTION
**Pull request by dankurka**
[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/141)

### CODE REVIEW

File: src/main/markdown/articles/gss_migration.md [hunk 1]
```diff
@@ -1 +1,199 @@
...
+This document outlines the steps one needs to take in order to move from CSS to
+GSS.
+
+Small applications want to use a one time migration, while big applications may
+want to use the Gradual migration path.
+
+# Configuration Properties
+
+## 
+Enable GSS
```
Comment (jDramaix) [inline src/main/markdown/articles/gss_migration.md]: bad formatting

File: src/main/markdown/articles/gss_migration.md [hunk 2]
```diff
@@ -1 +1,199 @@
...
+
+Enabling GSS will tell the GWT compiler to use GSS for all CssResource
+interfaces. This flag can either be “false” (default) or “true”.
+
+
+<pre class=prettyprint>
```
Comment (jDramaix) [inline src/main/markdown/articles/gss_migration.md]: replace all <pre class=prettyprint> by triple backticks

File: src/main/markdown/articles/gss_migration.md [hunk 3]
```diff
@@ -1 +1,199 @@
...
+&lt;set-configuration-property name="CssResource.enableGss" value="true" />
```
Comment (jDramaix) [inline src/main/markdown/articles/gss_migration.md]: by using triple backticks, you are not obliged to escape hml

File: src/main/markdown/articles/gss_migration.md [hunk 4]
```diff
@@ -1 +1,199 @@
...
+@Source({"def.css", "converted.gss"})
+MyCssResource css();
+</pre>
+
+  * For base types in the SDK  every css file has a gss equivalent.
+
+# Gradual migration
+
+  1. [Run the conversion tool on a certain subdirectory of your app, this will create
+a .gss file for ](#heading=h.1dt9dd9gim7n)every .css file.
```
Comment (jDramaix) [inline src/main/markdown/articles/gss_migration.md]: this link is invalid here.

File: src/main/markdown/articles/gss_migration.md [hunk 5]
```diff
@@ -1 +1,199 @@
...
+Since the converter needs to convert all variables to a GSS compatible format,
+it needs to be aware of all variables that are defined in scope. It therefore
+refuses to convert if it encounters an undefined variable.
+
+In order to get around this limitation the converter allows you to specify a
+list of css files to be parsed for building up scope:
+
+
+<pre class=prettyprint>
+$ /google/src/head/depot/google3/third_party/java/gwt/bin/Css2Gss -scope
```
Comment (jDramaix) [inline src/main/markdown/articles/gss_migration.md]: remove the path

File: src/main/markdown/articles/gss_migration.md (hunk 1 above)
Comment (dankurka) [inline src/main/markdown/articles/gss_migration.md]: Done

File: src/main/markdown/articles/gss_migration.md (hunk 2 above)
Comment (dankurka) [inline src/main/markdown/articles/gss_migration.md]: done

File: src/main/markdown/articles/gss_migration.md (hunk 3 above)
Comment (dankurka) [inline src/main/markdown/articles/gss_migration.md]: done

File: src/main/markdown/articles/gss_migration.md (hunk 4 above)
Comment (dankurka) [inline src/main/markdown/articles/gss_migration.md]: removed

File: src/main/markdown/articles/gss_migration.md (hunk 5 above)
Comment (dankurka) [inline src/main/markdown/articles/gss_migration.md]: done

**Pull request merged**